### DISCUSSION TITLE
Pointing Endpoint To Docker Socket Proxy

### DISCUSSION THREAD

**Discussion by luizmndz** [sablierapp/sablier#479] [Ideas]
Is it possible to point sablier to a docker socket proxy like https://github.com/Tecnativa/docker-socket-proxy?

**Comment by acouvreur** [comment]
Yes, you can use docker environment variables, and set the DOCKER_HOST to the proxy url.

**Reply by Finch290** [reply]
what variables sablier needs between 
    EVENTS
    PING
    VERSION
    AUTH
    SECRETS
    POST: 
    BUILD
    COMMIT
    CONFIGS
    CONTAINERS
    ALLOW_START (containers/id/start)
    ALLOW_STOP (containers/id/stop)
    ALLOW_RESTARTS (containers/id/stop|restart|kill)
    DISTRIBUTION
    EXEC
    GRPC
    IMAGES
    INFO
    NETWORKS
    NODES
    PLUGINS
    SERVICES
    SESSION
    SWARM
    SYSTEM
    TASKS
    VOLUMES
thank you!
